.github/workflows/tests.yml

```diff
@@ -1,27 +1,45 @@
-name: CI:Tests
+name: Tests
 
 # Run on any commit or pr on any branch
 on:
   pull_request:
     branches:
       - '**'
 
+
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
   # This workflow contains a single job called "build"
   build:
     runs-on: ubuntu-latest
-    
+
+
     steps:
     # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
     - uses: actions/checkout@v2
 
+    - uses: actions/cache@v2
+      name: Cache website modules
+      with:
+        path: |
+          ./website/node_modules
+        key: ${{ runner.os }}-website-modules-${{ hashFiles('./website/yarn.lock') }}
+
     - name: Run Website Unit Tests
       run: |
          cd website 
          yarn 
          yarn test
 
+    - uses: actions/cache@v2
+      name: Cache database modules and firebase binary
+      with:
+        path: |
+          ./Firebase/Tests/node_modules
+          ${{ github.workspace }}/emulator-cache
```

Comment:
Since the cache did not hit in the runner: 
```
⚠  It appears you are running in a CI environment. You can avoid downloading the Firestore Emulator repeatedly by caching the /home/runner/.cache/firebase/emulators directory.
```

Reply:
I switched out that path, and I'm no longer getting that message in the logs. It appears to be caching correctly now

Reply:
I'm not sure where the button is to "approve" your changes or mark them as done, but I added them to this PR